src/frontend/context/EndretUtbetalingAndelContext.ts

```diff
@@ -188,7 +190,7 @@ const [EndretUtbetalingAndelProvider, useEndretUtbetalingAndel] = createUseConte
             kanSendeSkjema,
             onSubmit,
             hentSkjemaData,
-            tilbakestillFelterTilDefault,
+            tilbakestillFelterTilDefault: settFelterTilDefault,
```

Comment:
Jeg beholdt det eksponerte navnet da jeg lurer på om tilbakestill gir mening utenfor konteksten. Men veldig åpen for å endre her også. Også veldig åpen for å endre på navnet på funksjonen også

Reply:
Syns det er greit å ha samme navn innad i komponenten som det som er eksponert ut, men både `tilbakestillFelterTilDefault` og `settFelterTilDefault` fungerer for meg